### PR TITLE
feat: adding header support for Producer.send

### PR DESCRIPTION
**Pull request by danielbatica**
- fixed also the Native Kafka Client producer send ( opaqueKey is not used by the dependency)
- updated dependency with versions that send headers as well

Co-authored with @solaris007

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -69,7 +69,8 @@
         "lodash.clonedeep": "~4.5.0",
         "most": "1.7.3",
         "most-subject": "5.3.0",
-        "sinek": "~9.0.0",
+        "node-rdkafka": "^2.10.1",
```
Comment (krystianity) [inline package.json:72]: @rob3000 probably the right thing to do until JS client api is fully supported one day

**Review by krystianity — COMMENTED** [review]
just checking with rob

File: package.json (hunk 1 above)
Comment (rob3000) [reply package.json:72]: Sounds good to me!

**Comment by danielbatica** [comment]
@rob3000 , @krystianity Do you guys plan on merging this in the near future? Thank you!

**Review by rob3000 — APPROVED** [review]

**Pull request merged**